src/kernel/gdt/gdt.c

```diff
@@ -0,0 +1,83 @@
+#include "gdt.h"
+#include <std/memory.h>
+
```

Comment:
I'm unfamiliar with OSWiki's code licensing but this is copied so something must be said, even if the commit message and the header make a mention of the source.

Reply:
wiki.osdev.org is public domain. Added a note for where the implementation is derived from in 
72c3567.